reference/upgrades.md

```diff
@@ -1,5 +1,96 @@
 ## Upgrading Mayastor
 
-Mayastor v2.0.0 does not support upgrade from previous versions. The installation of the new version should be preceded by the removal of any earlier version.
+Upgrade to the latest Mayastor version is supported only from 2.1.0 or later. To upgrade from a previous version, visit [Manual steps to upgrade Mayastor](manual-steps-to-upgrade-mayastor). 
+
+### Supported upgrade paths
+
+- From 2.0.x to 2.1.0
+
+{% hint style="info" %}
+The process of upgrade utilises the [Mayastor Kubectl Plugin](https://mayastor.gitbook.io/introduction/advanced-operations/kubectl-plugin) and is non-disruptive only for volumes with more than one healthy replicas.
+{% endhint %}
+
+To upgrade all the Mayastor components of a cluster, execute:
+
+{% tab title="Command" %}
+```text
+kubectl mayastor upgrade
+```
+{% endtab %}
+
+ Mayastor 2.1.0 has the capability to perform selective-component upgrades. To view the flags that can be passed in order to achieve a non-disruptive selective upgrade, execute the following command:
```

Comment:
Typo - an extra space before Mayastor 2.1.0
The sentences are not providing clarity - what do we mean by perform selective-component upgrades? What do we mean by non-disruptive selective upgrade?

Reply:
By selective upgrade I meant the users can now pass flags like `--skip-data-plane-restart` ,  `--skip-single-replica-volume-validation` and now upgrade all the components at once. Is there another word for it that's we'd like in the user documentation? @avishnu

Reply:
> By selective upgrade I meant the users can now pass flags like `--skip-data-plane-restart` , `--skip-single-replica-volume-validation` and not necessarily upgrade all the components at once. Is there another word for it, that we'd like in the user documentation? @avishnu

I think we are still upgrading the components. I think, we should not "stress" much on the --skip_<> options, because they are not the recommended steps.
@sinhaashish @niladrih , wdyt ?

Reply:
Yes, those are very rebellious ways of upgrading, typically we have no room for selective upgrade.

Reply:
To explain further, we should be documenting the various possible options from the plugin. However, these are not really recommended ones, so we should not highlight them as a feature. Eg. The current text talks about "selective upgrade" as if it's an important feature, but it's not in reality.